app/components/primer/alpha/select_panel.rb

```diff
@@ -388,7 +388,10 @@ def initialize(
         @title = title
         @system_arguments[:tag] = :"select-panel"
         @system_arguments[:preload] = true if @src.present? && preload?
-
+        @system_arguments[:classes] = class_names(
```

Comment:
not sure if this is necessary since we're not actually using this class for styles, but thought it felt like good best practice, but happy to take it out if it is more misleading.